### PR TITLE
feat: support chatting in center peek

### PR DESCRIPTION
**Pull request by graydawnc**

### CODE REVIEW

**Review by pengx17 — COMMENTED** [review]
Any advice on how to review this PR?

**Comment by graydawnc** [comment]
> Any advice on how to review this PR?

Maybe can review if there is any logic that seems strange, or the component implementation is unreasonable?

File: packages/frontend/core/src/blocksuite/presets/ai/chat-panel/index.ts [hunk 1]
```diff
@@ -122,6 +122,7 @@ export class ChatPanel extends WithDisposable(ShadowlessElement) {
         this._chatSessionId = history.sessionId;
         this.chatContextValue.chatSessionId = history.sessionId;
         items.push(...history.messages);
+        AIProvider.LAST_ROOT_SESSION_ID = history.sessionId;
```
Comment (EYHN) [inline packages/frontend/core/src/blocksuite/presets/ai/chat-panel/index.ts:125]: why?

**Review by EYHN — APPROVED** [review]

File: packages/frontend/core/src/blocksuite/presets/ai/chat-panel/index.ts (hunk 1 above)
Comment (graydawnc) [reply packages/frontend/core/src/blocksuite/presets/ai/chat-panel/index.ts:125]: Each doc will have one and only one root chat session, which is the session corresponding to the conversation in the right sidebar. It is needed when conducting conversations or forking sessions. This session id needs to be recorded when loading the sidebar for the first time or switching doc.

**Pull request merged**